charts/mi-adf-shir/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: mi-adf-shir
 home: https://github.com/hmcts/mi-adf-integration-runtime/
-version: 1.1.2
+version: 1.10.3
```

Comment:
1.10.3 because previous mistake releasing 1.10.0 is causing renovate to think that was the latest.